packages/server/modules/serverinvites/tests/invites.spec.js

```diff
@@ -409,7 +410,7 @@ describe('[Stream & Server Invites]', () => {
         // Creating some invites
         await Promise.all(
           invites.map((i) =>
-            createInviteDirectly(i, me.id).then((o) => {
+            createInviteDirectly({ db: knexInstance })(i, me.id).then((o) => {
```

Comment:
As mentioned in Discord, we should try managing reusable dependency objects so that we don't have to keep repeating these same injections all the time.